docs/source/faq.rst

```diff
@@ -31,10 +34,53 @@ configured some templates to help and guide you through the process and
 hopefully to make it easier for you.
 
 Before submitting your bug, you can `search existing issues`__ to see if there
-is one open already.
+is one open already. If no existing issue covers your bug, please :ref:`have log
+excerpts ready <faq-logging>` that demonstrate the problem.
 
-Do not hesitate to :ref:`contact us<faq-contact-us>` if you are unsure about
+Do not hesitate to :ref:`contact us <faq-contact-us>` if you are unsure about
 your bug report.
 
 .. __: https://github.com/sopel-irc/sopel/issues/new/choose
 .. __: https://github.com/sopel-irc/sopel/issues
+
+
+.. _faq-logging:
+
+How to obtain logs?
+===================
+
+When requesting help, you'll probably be asked to provide logs illustrating the
+problem. If you :ref:`report a bug <faq-bug-report>`, the form includes a field
+for "Relevant logs" that you should fill in if possible.
+
+By default you will find Sopel's logs in the ``~/.sopel/logs`` folder. If Sopel's
+config file specifies the :attr:`~.config.core_section.CoreSection.homedir`
+and/or :attr:`~.config.core_section.CoreSection.logdir` setting(s), you will find
+the log files in ``/path/to/homedir/logdir``.
+
+Log files' names start with the name of their associated config file, i.e. the
+:option:`--config <sopel start --config>` argument passed when starting Sopel.
+
+There are two main log types you might be asked to provide:
+
+* ``config_name.sopel.log``: Everything Sopel does from startup to shutdown.
+  In normal operation, the ``INFO`` (default) or ``WARNING``
+  :attr:`~.config.core_section.CoreSection.logging_level` is sufficient, but you
+  might be asked to enable ``DEBUG`` logging and reproduce your problem to help
+  Sopel's developers determine what's happening.
+* ``config_name.raw.log``: Raw traffic between Sopel and the IRC server. Not
+  enabled by default; turn it if needed on using the
+  :attr:`~.config.core_section.CoreSection.log_raw` setting in your config file.
+
+.. warning::
+
+    The ``raw`` log may contain sensitive information, e.g. your bot's NickServ
+    account credentials, its IP address, or channel keys.
+
+    Always check the portion of your ``raw`` log that you are sharing for
+    secrets, and censor any you find!
```

Comment:
I appreciate the treatment of the distinct flavors of logs here! However, I think we should have a cleaner separation between the two, so that the canonical answer to _"how do I get my logs?"_ is _"find `~/sopel/logs/whatever.sopel.log` or equivalent"_ and we have some supplementary information about how to get the raw stuff should it be needed.

Reply:
What exactly are you suggesting? A second FAQ titled (e.g.) "How to obtain raw logs?"

Or maybe reformatting this to be less bullet-y, maybe using subheadings instead?

Reply:
Basically, push the stuff that's talking about raw logs _further_ away from the path most users are going to take. Subheadings would be good, with the regular logs first.